src/rfc5322.rs

```diff
@@ -152,7 +151,7 @@ impl<'s> Rfc5322Parser<'s> {
     }
 
     /// Consume an unstructured from the input.
-    #[unstable]
+    /// [unstable]
```

Comment:
unstable-markers are deprecated, so I converted them to docstrings to keep the info (for API review).